synfig-studio/src/gui/mainwindow.cpp

```diff
@@ -194,6 +194,22 @@ MainWindow::init_menus()
 		sigc::ptr_fun(App::edit_custom_workspace_list)
 	);
 
+	//animation tabs
+	std::string arg_1[9] = {"move-to-tab-last","move-to-tab-1","move-to-tab-2","move-to-tab-3",
```

Comment:
I think this part can be improved. Something like this:
```
for (int i = 1; i <= 8; ++i) {
	const std::string tab = std::to_string(i);
	action_group->add(Gtk::Action::create("move-to-tab-" + tab, _("Move to Tab ") + tab),
		[this, i]() { main_dock_book().set_current_page(i-1); }
	);
}
action_group->add(Gtk::Action::create("move-to-tab-last", _("Move to Last Tab")),
	[this]() { main_dock_book().set_current_page(-1); }
);
```

Reply:
I actually really like this it basically does everything at once (^_^).